src/spyglass/utils/dj_helper_fn.py

```diff
@@ -612,3 +644,4 @@ def str_to_bool(value) -> bool:
     if not value:
         return False
     return str(value).lower() in ("y", "yes", "t", "true", "on", "1")
+    return str(value).lower() in ("y", "yes", "t", "true", "on", "1")
```

Comment:
Why is on here? I would be worried it would be a common misspelling of no.

Reply:
I think that's on me for pulling from this `PostHog/posthog` repo that covered all bases by including `on` vs `off`. The edit makes sense to me